ta/os_test/os_test.c

```diff
@@ -623,7 +623,7 @@ static TEE_Result test_time(void)
 	}
 	MSG("TA time %"PRIu32".%03"PRIu32, t.seconds, t.millis);
 
-	if (t.seconds > 1) {
+	if (t.seconds > 60) {
```

Comment:
Test at line 603 is not impacted by the same CPU overload issue?

Reply:
I don't know, it's at least far less likely since we're not exiting to the normal world in between.

Reply:
Maybe fragile, depends on the scheduler. Won't this trigger rare CI test failures?

Reply:
Sure, I'll increase the time there too.